package.toml

```diff
@@ -0,0 +1,2 @@
+[buildpack]
+uri = "."
```

Comment:
This is the default value for the file, so this whole file can be removed if preferred.

Reply:
This file is what cutlass is using to know whether you've specified a valid dir or not https://github.com/heroku/cutlass/blob/313bf47448011863a77acccb380502d9953c69cb/lib/cutlass/local_buildpack.rb#L74